test/tests/meshgenerators/lower_d_block_generator/tests

```diff
@@ -26,22 +26,13 @@
                   "existing sideset for"
     issues = '#13814'
     design = Mesh/index.md
-
-    [first_order]
-      type = Exodiff
-      exodiff = 'lower_d_out.e'
-      input = 'lower_d.i'
-      mesh_mode = REPLICATED
-
-      detail = "first-order and"
-    []
```

Comment:
If we are deleting this test, I think the requirement on line 5 should be added to, something like
```
The system shall be able to create different subdomains with first order elements
```

Reply:
Actually maybe this is not necessary as it perhaps implies that you can only do it for first order elements (even though the next requirement will show otherwise)